index.ts

```diff
@@ -2,7 +2,7 @@ import { getRuntime } from './lib/utils/Utils';
 
 // Polyfill fetch for node
 if (getRuntime() === 'node') {
-  const undici = require('undici');
+  const undici = Reflect.get(module, 'require')('undici');
```

Comment:
This does not work when you try to require/import YouTube.js externally, it throws `Error: Cannot find module 'undici'` even though undici is in fact present in `node_modules`. Changing it back to `require('undici')` works just fine.

Reply:
Reverted 👍